docs/source/tracking/artifacts-stores.rst

```diff
@@ -0,0 +1,200 @@
+===============
+Artifact Stores
+===============
+
+The artifact store is a core component in `MLflow Tracking <../index.html>`_ where MLflow stores (typicaly large) arifacts
+for each run such as model weights (e.g. a pickled scikit-learn model), images (e.g. PNGs), model and data files (e.g. `Parquet <https://parquet.apache.org/>`_ file). 
+Note that metadata like parameters, metrics, and tags are stored in `backend store <backend-stores.html>`_, the other component of the MLflow Tracking.
+
+Configure Artifact Store
```

Comment:
```suggestion
Configuring an Artifact Store
```
I think going with the gerund form fits in nicely with your other h2 headings below, as well as the instructional style of content